notebooks/official/tensorboard/vertex_tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<p>"AI Platform" is a legacy term. Use "Vertex AI" instead. In this case,  it should be "Vertex AI Training Service"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>